package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "pagopa-arc-fe",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/pagopa/pagopa-arc-fe.git",
+  "author": "anislucidi01 <anis.lucidi@pagopa.it>",
+  "license": "MIT",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix",
+    "format": "prettier --write './**/*.{js,jsx,ts,tsx,css,md,json}' --config ./.prettierrc"
+  },
+  "devDependencies": {
+    "@babel/plugin-transform-modules-commonjs": "^7.24.1",
+    "autochangelog": "^3.0.3",
+    "danger": "^11.3.1",
+    "danger-plugin-digitalcitizenship": "^1.0.0",
+    "dotenv": "^16.4.5",
```

Comment:
How "danger" and "danger-plugin-digitalcitizenship"relates to prettier, eslint, typescript?

Reply:
should I delete it for now?

Reply:
I see configuration files for eslint, prettier and typescript.  
I think that only libraries that relates to these tools should be present in the package.json at this stage 
I would prefer the cleanest starting point possible.